observability/Makefile

```
@@ -0,0 +1,9 @@
+
```

(nit) so you can run `make` or `make all` and it will do everything for you
```suggestion
all: deploy patch-frontend patch-clusterservice
```